pcbdraw/populate.py

```diff
@@ -40,8 +48,17 @@ def enable_pcbdraw(self) -> None:
             r"([\s\S]+?\|[\s\S]+?)"   # side| component
             r"\]\](?!\])"             # ]]
         )
-        self.rules.insert(3, 'pcbdraw')
-        self.register_rule('pcbdraw', pcbdraw_pattern, parse_pcbdraw)
+        if mistune2:
+            self.rules.insert(3, 'pcbdraw')
+            self.register_rule('pcbdraw', pcbdraw_pattern, parse_pcbdraw)
+        else:
+            self.rules.pcbdraw = re.compile(pcbdraw_pattern)
+            self.default_rules.insert(3, 'pcbdraw')
+
+    def output_pcbdraw(self, m: re.Match[str]) -> Any:
+        _, side, components = parse_pcbdraw(self, m)
+        return self.renderer.pcbdraw(side, components)
```

Comment:
Please, add a comment explaining that this is here only for mistune 0.8

Reply:
Ok, added in 8b2158d